src/qibolab/designs/channels.py

```diff
@@ -35,19 +35,26 @@ class Channel:
     Quantum Machines associate filters to channels but this may not be the case
     in other instruments.
     """
+    max_bias: Optional[float] = None
+    """Maximum voltage that we can send for flux bias without damaging amplifiers.
+    If the user attempts to send a higher value the platform will raise an error
+    to avoid the operation of being executed in the real instruments.
+    """
 
     @property
     def bias(self):
         """Bias offset for flux channels."""
         if self._bias is None:
             # operate qubits at their sweetspot unless otherwise stated
-            return self.qubit.sweetspot
+            self.bias = self.qubit.sweetspot
```

Comment:
Why is this an infinite recursion loop? I think it is just calling the setter, not the property, but it may not be good practice, I am not sure.

The reason I am doing this here, is because I want the 

https://github.com/qiboteam/qibolab/blob/0bec714998aa303e281d939eae26a70c336a7672/src/qibolab/designs/channels.py#L56

check to run before setting the bias as the sweetspot value. Otherwise I will have to repeat the check both here and in the setter.

Reply:
However, choose the solution you prefer, but I'd suggest to you to split the condition, i.e.
https://github.com/qiboteam/qibolab/blob/0bec714998aa303e281d939eae26a70c336a7672/src/qibolab/designs/channels.py#L56
and give it a name. I.e. make a function of it.

I know it might seem silly to make such a short function, but:
- giving it a name it will be more explicit
- in principle you can also include the error, not only the condition
  https://github.com/qiboteam/qibolab/blob/0bec714998aa303e281d939eae26a70c336a7672/src/qibolab/instruments/qm.py#L771-L775

So you will have:
```py
def check_max_bias(max_value, max_bias):
    if max_bias is not None and max_value > max_bias:
        raise_error(
            ValueError,
            f"Cannot sweep bias up to {max_value} because the maximum supported value is {max_bias}.",
        )
```

And in this way you don't even have to repeat the error, and will be even consistent in `channels.py` and `qm.py` :)